themes/SuiteP/css/Dawn/variables.scss

```diff
@@ -939,6 +939,8 @@ $default-btn-bg-hover: $color-28;
 $text-hover-listview: $color-80;
 $subpanel-text-hover: $color-90;
 
+$text-color-white: $color-90;
```

Comment:
Could we change the name of this variable to remove the explicit color? it's possible that ```$color-90``` might be changed to be something other than white.

It would also be good to squash the 4 commits here down into one if we can.